src/USER-CAC/asa_user.h

```diff
@@ -0,0 +1,388 @@
+
+#ifndef ASA_USER_H
+#define ASA_USER_H
+#include <limits.h>
```

Comment:
please use C++ style include files for C library headers and try to utilize settings from lmptype.h as much as possible.